android-json-form-wizard/src/test/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactoryTest.java

```diff
@@ -180,6 +180,17 @@ public void testUniqueChildElementKeyGenerationShouldContainChildKeyAsComponent(
         Assert.assertEquals(childElementKey, uniqueIdComponents[0]);
     }
 
+    @Test
+    public void testRepeatingGroupDefaultCount() throws Exception {
+        View rootLayout = invokeGetViewsFromJson().get(0);
+
+        JSONObject repeatingGroupCountObj = step.getJSONArray(JsonFormConstants.FIELDS).getJSONObject(0);
+        Assert.assertEquals("0", repeatingGroupCountObj.getString(JsonFormConstants.VALUE));
+
+        ((MaterialEditText) rootLayout.findViewById(R.id.reference_edit_text)).setText("");
```

Comment:
Could you add an intermediate step where you set the text to something else and verify that that value is updated on `repeatingGroupCountObj`,  then set it to blank and verify that it's reset to `0`?

Reply:
added.